docs/install/docker.md

```diff
@@ -13,7 +13,7 @@ The recommended and easiest way to get started is with Docker. To learn more abo
 
 A sample [docker-compose.yaml](https://github.com/philosowaffle/peloton-to-garmin/blob/master/docker-compose.yaml) file and [configuration.local.json](https://github.com/philosowaffle/peloton-to-garmin/blob/master/configuration.example.json) can be found in the project repo.
 
-The Docker container expects a valid `configuration.local.json` file is mounted into the container.  Additionally, you can mount the `app/working` and `app/output` directories.  You can learn more about the configuration file over in the [Configuration Section](/{{ site.baseurl }}{% link configuration/index.md %}).
+The Docker container expects a valid `configuration.local.json` file is mounted into the container.  Additionally, you can mount the `app/working` and `app/output` directories.  You can learn more about the configuration file over in the [Configuration Section](https://philosowaffle.github.io/peloton-to-garmin/configuration/)

```

Comment:
```suggestion
The Docker container expects a valid `configuration.local.json` file is mounted into the container.  Additionally, you can mount the `app/working` and `app/output` directories.  You can learn more about the configuration file over in the [Configuration Section]({{ site.baseurl }}{% link configuration/index.md %})

```

The original URL syntax is correct as the site is compiled by Jeckyll which will replace the variable `site.baseurl` with the appropriate value (see line 34 below where the url works as expected).  I think the real problem here might be the leading `/`, causing the generated link to be broken.